config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  UNIREP_ADDRESS: '0x51714b0b1bE71464e6382108DE1F5398BaDe807C',
+  APP_ADDRESS: '0x87Aeb1Ff3E9b8AcF708a41A93832C704546606Cf',
+  ETH_PROVIDER_URL:
+    'https://arb-goerli.g.alchemy.com/v2/hfFfXlX8rR8YvrALiJ8b7ZtIPRGY1GTM',
+  PRIVATE_KEY:
+    '0x0f70e777f814334daa4456ac32b9a1fdca75ae07f70c2e6cef92679bad06c88b',
```

Comment:
added. should `config.mjs` just read these env variables? it currently is applying a ternary expression between the process variable and a string.

Reply:
I think we don't need this here?
it can just read the `.env` file
so when developers want to run this locally
they have to copy the env and input the right `ETH_PROVIDER_URL` and `PRIVATE_KEY`

and we can automatically build this in testnet and copy the `.env.example`